apps/schema/src/components/NomnomDiagram.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div>
+    <InputCheckbox
+      id="displaySettings"
+      v-model="displaySettings"
+      :defaultValue="displaySettings"
+      :options="['attributes', 'external', 'vertical', 'ontologies']"
+    />
+    <div v-html="nomnomlSVG" class="bg-white" style="max-width: 100%"></div>
+  </div>
+</template>
+
+<script>
+import { renderSvg } from "nomnoml";
+import { InputCheckbox } from "molgenis-components";
+
+export default {
+  props: {
+    schema: {
+      type: Object,
+      required: true,
+    },
+  },
+  components: {
+    InputCheckbox,
+  },
+  data() {
+    return {
+      imgFullscreen: false,
+      displaySettings: [],
+    };
+  },
+  methods: {
+    nomnomColumnsForTable(table, tableName) {
```

Comment:
needs test case , hard to see what is being build

Reply:
what is added value of a test case? Result would be a test where I would compare a 'nomnom' text string agains this function. Moreover, this script has now functional problems because if this view fails the user flow is not impeded.